lib/newman_conway.py

```diff
@@ -7,4 +7,16 @@ def newman_conway(num):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+    if num < 1:
+        raise ValueError("Must be a positive number")
+    if num == 1:
+        return "1"
+
+    current_list = [0, 1, 1]
+
+    count = 3
+    while count <= num:
+        current_list.append(current_list[current_list[count - 1]] + current_list[count - current_list[count - 1]])
+        count += 1
+
+    return [str(item) for item in current_list]
```

Comment:
🤔 To pass the tests, you want to return the newman conway numbers as a string (ex. "0 1 1") instead of a list of strings